### PR TITLE
Get project in line with convention

### PR DESCRIPTION
**Pull request by pbrisbin**
- Yarn
- Jest
- CI

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,4629 +0,0 @@
-{
```
Comment (joris974) [inline package-lock.json:1]: If you are using yarn you shouldn't have a `package-lock.json` but you should have a `yarn.lock` file

File: package.json [hunk 2]
```diff
@@ -8,11 +8,11 @@
     "serverless": "^1.63.0"
   },
   "devDependencies": {
-    "mocha": "^7.0.1"
+    "jest": "^25.1.0"
```
Comment (joris974) [inline package.json]: we always fix version in the frontend as a best practice

File: src/lib/parse-component-email.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const {parseComponentEmail} = require('./parse-component-email.js')
```
Comment (joris974) [inline src/lib/parse-component-email.test.js:1]: any way you can use `import {parseComponentEmail} from './parse-component-email.js'` 
You can do that via Babel `babel-preset-es2015` to be able to interpret ES6
with `.babelrc` like this:
```
{
    "presets": ["es2015"]
}
```

File: package-lock.json (hunk 1 above)
Comment (pbrisbin) [reply package-lock.json:1]: Right, and that's what this PR does. Am I misunderstanding?

File: src/lib/parse-component-email.test.js (hunk 3 above)
Comment (pbrisbin) [reply src/lib/parse-component-email.test.js:1]: Can you say more about how to do this? I change to use `import` and added the `.babelrc` but `yarn test` still fails with syntax error. What other configuration is required? I'll need the tests to run and deployment as the Lambda function to work.

Comment (joris974) [reply src/lib/parse-component-email.test.js:1]: It is something we can do later if you want to merge this now

File: package-lock.json (hunk 1 above)
Comment (joris974) [reply package-lock.json:1]: sorry I thought this file was still there. Ignore me

**Review by joris974 — APPROVED** [review]

**Pull request merged**